leafsmart-front-end/src/components/Dashboard.js

```diff
@@ -35,27 +35,27 @@ const Dashboard = () => {
         <div className="flex p-8 w-100">
           <div>
             <div className="flex justify-between">
-            <Link to="#">
-              <AddFavourite
-                icon={<FaPlus size="40"/>}
-              />
-            </Link>
-            <Search onSearchChange={handleOnSearchChange}/>
+              <Link to="#">
+                <AddFavourite icon={<FaPlus size="40" />} />
+              </Link>
+              <Search onSearchChange={handleOnSearchChange} />
             </div>
             {cityName && <Events cityName={cityName} />}
           </div>
           {cityName && (
             <div className="ml-12">
-              <CurrentWeather coordinates={coordinates}/>
-              <ForecastWeather coordinates={coordinates} />
+              <div className=" border rounded-3xl p-4 mb-8 mt-4 w-486 bg-gradient-to-r from-Independence to-HeliotropeGray">
```

Comment:
Looks beauty! I think the border class is the only thing betrayin' us here, but that's easy enough to remove! :D